deployment/CP05MOAS-A6264_Deploy.csv

```diff
@@ -110,4 +110,18 @@ AR61,,AR61,,CP05MOAS-A6264-02-DOSTAN000,16,1,2021-11-04T18:34:00,2021-11-05T20:1
+AR66,,AR66,,CP05MOAS-A6264-00-ENG000000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-01-FLORTN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-02-DOSTAN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-03-CTDAVN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00527,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-04-NUTNRN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-05-ADCPAN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-06-PARADN000,17,1,2022-04-12T19:07:00,2022-04-13T17:51:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50267,39.91838,-70.9088,,,,across-shelf survey,,
+AR66,,AR66,,CP05MOAS-A6264-00-ENG000000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-01-FLORTN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-02-DOSTAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-03-CTDAVN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00527,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-04-NUTNRN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-05-ADCPAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-06-PARADN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50267,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
```

Comment:
looks good; maybe add 'along-shelf' in between abbreviated & mission for the second AUV deployment just for consistency.